Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM python:3.10.12-slim-buster
 
 RUN apt-get -y update && apt-get -y upgrade && \
-    apt-get install --no-install-recommends -y curl ncat git mime-support gcc libc-dev libpq-dev
+    apt-get install --no-install-recommends -y curl ncat git mime-support gcc libc-dev libpq-dev libmagic-dev
```

Comment:
Instead of `libmagic-dev` you should use `libmagic1`, see doc of `python-magic`

Reply:
Thanks for the hint, done